testutil/util.go

```diff
@@ -61,6 +64,30 @@ func (t *T) Override(dest, tmp interface{}) {
 	t.teardownActions = append(t.teardownActions, teardown)
 }
 
+func (t *T) CopyFile(src, dst string) {
+	from, err := os.Open(src)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer from.Close()
+
+	to, err := os.OpenFile(dst, os.O_RDWR|os.O_CREATE, 0666)
```

Comment:
Maybe use `ioutil.ReadFile` and `ioutil.WriteFile` would be produce shorter code.